### ISSUE TITLE
world issue

### ISSUE THREAD

**Issue by OM1CRONs** [Cooper-VR/saoStyleRPG#4]
most of the features work ok, but the world is a little empty, it could use a but more to feel more finished.

**Comment by MedinaVRC** [comment]
yes, this. The town looks nice enough but the rest of the map could use a bit of substance
